.github/workflows/deploy-to-control-plane-review.yml

```
@@ -1,85 +1,103 @@
+              ref: context.sha,
+              environment: 'review-app',
+              auto_merge: false,
+              required_contexts: []
+            });
+            return deployment.data.id;
+
       - name: Get PR HEAD Ref
         if: ${{ github.event_name == 'issue_comment' }}
         id: getRef
-        run: echo "PR_REF=$(gh pr view $PR_NUMBER --repo ${{ github.repository }} --json headRefName | jq -r '.headRefName')" >> $GITHUB_OUTPUT
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:27: Double quote to prevent globbing and word splitting

(shellcheck)